internal/filtering/resources/aws_security_group_rule.go

```diff
@@ -0,0 +1,21 @@
+/*
+ * © 2023 Snyk Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package resources
+
+const AWSSecurityGroupRule = "aws_security_group_rule"
+
+var AWSSecurityGroupRuleAllowedAttributes = []string{"security_group_id"}
```

Comment:
Maybe we could set that directly in `internal/filtering/allowlist.go`.
I'm not seeing a lot of value to create a file per resource type.